src/server.c

```diff
@@ -4071,6 +4071,14 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    if ((c->cmd->flags & CMD_INTERNAL) && !((c->flags & CLIENT_INTERNAL) || mustObeyClient(c))) {
+        sds err = sdsnew(NULL);
+        err = sdscatprintf(err, "unknown command '%.128s'", (char*)c->argv[0]->ptr);
```

Comment:
Lets make sure this has the exact same format as the unknown command message above.

Reply:
It indeed is, verified locally as well.

Reply:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/99696e1d-f16e-4525-8aa2-4ac6b9529ed7" />

Reply:
So where are the `with args beginning with:`?
Not sure it is that critical, but still.

Reply:
Didn't think it was critical as well. Added.